pyInterface/plotAngles.py

```diff
@@ -17,7 +18,7 @@ def getPermutations(topology):
 	permutations = {}
 	for key in keys:
 		permutations[key] = [[None, None] for x in range(topology.nmbDecayVertices())]
-	vertex_i = 0
+	vertexI = 0
```

Comment:
I thought we wanted to keep the `_i`/`i_` variable names for indices? PyLint is skipping this file anyway, is it not?